test/NosCore.Tests/Helpers/TestHelpers.cs

```diff
@@ -309,7 +312,7 @@ public ClientSession GenerateSession()
                 Titles = new List<TitleDto>()
             };
             var charaDto = chara.Adapt<CharacterDto>();
-            CharacterDao.InsertOrUpdate(ref charaDto);
+            charaDto = await CharacterDao.TryInsertOrUpdateAsync(charaDto).ConfigureAwait(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'charaDto'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)